pages/login.js

```diff
@@ -1,11 +1,147 @@
-import React, {Component} from 'react';
-
+import React, { Component } from 'react';
+import style from './styles/login.module.scss';
+import { Button, Form, Icon } from 'semantic-ui-react';
+import Animation from '../components/Animation';
+import Particles from 'react-particles-js';
 class Login extends Component {
     render() {
         return (
-            <>
-                <p> Hi </p>
-            </>
+            <div className={style.container}>
+                <div className={style.left}>
+                    <Particles className={style.left} params={{
```

Comment:
rolf, clean up the params, it's too long.

If you wanna use the setting, you should import it into a separate JSON file and import it back.

use 

`import * as paramsData from './somewhere.json'`